deploy/examples/create-external-cluster-resources.py

```diff
@@ -762,21 +747,26 @@ def get_active_and_standby_mgrs(self):
 
         try:
             monitoring_endpoint_ip = self._convert_hostname_to_ip(
-                monitoring_endpoint_ip
+                monitoring_endpoint_ip, monitoring_endpoint_port
             )
             # collect all the 'stand-by' mgr ips
             mgr_ips = []
             for each_standby_mgr in standby_mgrs:
                 failed_ip = each_standby_mgr
-                mgr_ips.append(self._convert_hostname_to_ip(each_standby_mgr))
+                mgr_ips.append(
+                    self._convert_hostname_to_ip(
+                        each_standby_mgr, monitoring_endpoint_port
```

Comment:
The port for standby_mgr would be also same, right?

Reply:
Yes, i'd expect the port to be the same between active and standby mgr